CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.1.2](https://github.com/ASFHyP3/hyp3-insar-gamma/compare/v2.1.1...v2.1.2)
+
+### Fixed
```

Comment:
I like to call out hyp3 lib version updates in the changelog, e.g. https://github.com/ASFHyP3/hyp3-rtc-gamma/blob/develop/CHANGELOG.md#230 , but not blocking